packages/openneuro-app/src/scripts/refactor_2021/search/use-search-results.tsx

```diff
@@ -285,6 +286,7 @@ export const useSearchResults = () => {
     errorPolicy: 'ignore',
     // fetchPolicy is workaround for stuck loading bug (https://github.com/apollographql/react-apollo/issues/3270#issuecomment-579614837)
     // TODO: find better solution
-    fetchPolicy: 'no-cache',
+    fetchPolicy: 'cache-and-network',
+    nextFetchPolicy: 'cache-first',
```

Comment:
I think as long as we're using the anonymous SSR -> authenticated client renders this is the right strategy but we might want to revisit if we re-enable authenticated SSR to save a query.